tests/v2/validation/rbac/snapshotrbac/snapshot_rbac_test.go

```diff
@@ -99,11 +106,22 @@ func (etcd *SnapshotRBACTestSuite) TestRKE2K3SSnapshotRBAC() {
 			require.NoError(etcd.T(), err)
 			clusterClient = relogin
 
-			etcd.testRKE2K3SSnapshotRBAC(tt.role, clusterClient)
+
+		if !(strings.Contains(etcd.cluster.ID, "c-m-")) {
+			etcd.testRKE1SnapshotRBAC(tt.role, clusterClient)
+			subSession := etcd.session.NewSession()
+			defer subSession.Cleanup()
+		} else {
+				etcd.testRKE2K3SSnapshotRBAC(tt.role, clusterClient)
+				subSession := etcd.session.NewSession()
+				defer subSession.Cleanup()
+			}
```

Comment:
This can be modified to:
```
		if !(strings.Contains(etcd.cluster.ID, "c-m-")) {
			etcd.testRKE1SnapshotRBAC(tt.role, clusterClient)
		} else {
				etcd.testRKE2K3SSnapshotRBAC(tt.role, clusterClient)
			}
		subSession := etcd.session.NewSession()
		defer subSession.Cleanup()
```